rclcpp/src/rclcpp/node_interfaces/node_graph.cpp

```diff
@@ -163,10 +183,19 @@ NodeGraph::get_node_names() const
       std::string("could not destroy node names: "));
     // *INDENT-ON*
   }
+  ret = rcutils_string_array_fini(&node_namespaces_c);
+  if (ret != RCUTILS_RET_OK) {
+    // *INDENT-OFF*
+    // TODO(karsten1987): Append rcutils_error_message once it's in master
```

Comment:
I don't really understand this TODO (copied here or from the original) @Karsten1987 can you explain?